### PR TITLE
Add server daemonset yaml template files

### PR DESCRIPTION
**Pull request by sufuf3**
- Add tcp & unix with namespace vortex
- Fake image name

---

- [x] tcp can run
- [x] unix can run

test files are under `~/go/src/github.com/linkernetworks/network-controller/example/server-yaml`

### CODE REVIEW

**Comment by sufuf3** [comment]
I added the following config in these yaml files.
```yaml
        securityContext:
          privileged: true
```

Could you please help me review this PR, again?
Many thanks.

File: deploy/server/unix-daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      - name: network-controller-server-unix
+        image: linker/net-controller-server:latest
+        securityContext:
+          privileged: true
+        command: ["./network-controller-server"]
+        args: ["-unix=/tmp/a.sock"]
+        volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker-sock
+        - mountPath: /tmp/a.sock
```
Comment (John-Lin) [inline deploy/server/unix-daemonset.yaml]: rename to `/var/run/vortex.sock`

File: deploy/server/unix-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          name: grpc-sock
+      volumes:
+      - name: docker-sock
+        hostPath:
+          path: /var/run/docker.sock
+      - name: grpc-sock
+        hostPath:
+          path: /tmp/a.sock
```
Comment (John-Lin) [inline deploy/server/unix-daemonset.yaml]: rename to /var/run/vortex.sock

File: deploy/server/unix-daemonset.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: network-controller-server-unix
+        image: linker/net-controller-server:latest
+        securityContext:
+          privileged: true
+        command: ["./network-controller-server"]
+        args: ["-unix=/tmp/a.sock"]
```
Comment (John-Lin) [inline deploy/server/unix-daemonset.yaml]: rename to `/var/run/vortex.sock`

File: deploy/server/tcp-daemonset.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: network-controller-server-tcp
+  namespace: vortex
+  labels:
```
Comment (John-Lin) [inline deploy/server/tcp-daemonset.yaml]: At this moment. we don't need this label

File: deploy/server/unix-daemonset.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: network-controller-server-unix
+  namespace: vortex
+  labels:
```
Comment (John-Lin) [inline deploy/server/unix-daemonset.yaml]: At this moment. we don't need this label

**Comment by sufuf3** [comment]
I removed labels. Also, updated the socket name. 
Mount path need to be setup folder, so I use `/var/run` as `volumes.hostPath.path` & `volumeMounts.mountPath`.

Please help me review this PR, again.
Thank you very much.

File: deploy/server/unix-daemonset.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+          name: docker-sock
+        - mountPath: /var/run/
+          name: grpc-sock
+      volumes:
+      - name: docker-sock
+        hostPath:
+          path: /var/run/docker.sock
+      - name: grpc-sock
+        hostPath:
+          path: /var/run/
```
Comment (John-Lin) [inline deploy/server/unix-daemonset.yaml]: I found mount entire `/var/run/` can cause significant security issue. we should change back to `/tmp`

Comment (John-Lin) [reply deploy/server/unix-daemonset.yaml]: Unix sockets only live while the program is running, so `/tmp/` is usually an alright place for them to live

**Pull request merged**